client/src/services/courses.js

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import axios from './axios';
+
+const getCourses = async () => {
+  const response = await axios.get('/v1/user/courses');
+  console.log(response.data);
+  return response.data;
+};
```

Comment:
Should there be a try catch here too?

Reply:
I think that try catch is not required here if it's included in the FrontPage component. If the get is not successful, it will return 400 or some other error which will be handled in the try catch within the component.

Reply:
Okay, working on these and adding catch() to useEffect().